api/users/views.py

```diff
@@ -401,8 +405,48 @@ def get_default_odm_query(self):
             Q('is_registration', 'eq', True) &
             Q('contributors', 'eq', user._id)
         )
+
         permission_query = Q('is_public', 'eq', True)
         if not current_user.is_anonymous():
             permission_query = (permission_query | Q('contributors', 'eq', current_user._id))
         query = query & permission_query
         return query
+
+class UserNodeLogs(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin, UserMixin):
```

Comment:
You need to put documentation on this endpoint so users understand what it is returning, and the fact you can add an aggregate query.